docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -50,6 +50,42 @@ The overload manager uses Envoy's :ref:`extension <extending>` framework for def
 resource monitors. Envoy's builtin resource monitors are listed
 :ref:`here <v3_config_resource_monitors>`.
 
+.. _config_overload_manager_cgroup_memory:
+
+Cgroup Memory
+^^^^^^^^^^^^
+
+The cgroup memory resource monitor tracks memory pressure by reading memory usage and limits from the cgroup memory subsystem.
+This monitor supports both cgroup v1 and v2.
+
+The monitor reports memory pressure as a ratio of current memory usage to the memory limit. The limit is determined by:
+
+1. The configured max_memory_bytes value if specified, or
+2. The cgroup memory limit (from memory.limit_bytes for v1 or memory.max for v2) if max_memory_bytes is not configured
```

Comment:
Can you add to this how the system works if the two conflict `max_memory_bytes` and `cgroup memory limit`?